backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/webservice/dataloader/PhysicalStoreByStoreIdLoader.kt

```diff
@@ -11,7 +11,7 @@ import org.jetbrains.exposed.sql.transactions.transaction
 val physicalStoreByStoreIdLoader = newNamedDataLoader<Int, _>("PHYSICAL_STORE_BY_STORE_ID_LOADER") { ids ->
     transaction {
         PhysicalStoreEntity.find { PhysicalStores.storeId inList ids }
-            .sortByKeys({ it.storeId }, ids)
+            .sortByKeys({ it.storeId.value }, ids)
```

Comment:
A bit annoying that Kotlin did not catch this as a type error... Probably the generic was resolved to some common super type.